src/gui/standalone.ts

```diff
@@ -0,0 +1,27 @@
+import { ide } from "./ide";
+import { tgui } from "./tgui";
+
+export function handleTurtle(){
+    tgui.releaseAllHotkeys();
+    ide.turtle.parentNode.removeChild(ide.turtle);
+    document.body.innerHTML = "";
+    document.body.appendChild(ide.turtle);
+    ide.turtle.style.width = "100vh";
+    ide.turtle.style.height = "100vh";
+}
+
+
+export function handleCanvas(){
+    tgui.releaseAllHotkeys();
+    let cv = ide.canvas.parentNode;
+    cv.parentNode.removeChild(cv);
+    document.body.innerHTML = "";
+    document.body.appendChild(cv);
+    cv.style.width = "100vw";
+    cv.style.height = "100vh";
+    cv.style.top = "0px";
+    setTimeout( ()=>{
+        cv.width = 250;
+        cv.height = 250;
+    }, 100);
```

Comment:
This doesn't seem to be ported correctly. In exports, the canvas gets a size of 0x0...

<img width="728" alt="Canvas Export" src="https://user-images.githubusercontent.com/28811733/106260987-55ed7400-6221-11eb-8fa4-e2b82e7c7cd3.png">

This is how it was done previously:

https://github.com/TGlas/tscript/blob/88af6a8e64703f7c25120f49406bf5a7bc4be62c/examples/demos/cube3D.html#L10645-L10646

Reply:
On Chrome Version 88.0.4324.104 x64 the previous way dose not work.  It will set the canvas to have no size. This is also the case for this repo. So i simply changed it to use a 250^2 canvas. I dont see how my code could lead to the canvas havin no size.

As to why the size needs to be adjusted, the code of the ide will adjust the size of the canvas if the container changes size, this happens when the body is cleared.

I dont see how i could fix that without chaining a lot and it seems that this behavior is browser dependent....
Seeing as fixing this is not the goal of this PR i would suggest adding a new issue etc.

Reply:
Part of the problem might be that you are using e.h. `cv.height` to set the height of the _canvas_. However, `cv` refers to the containing `div` element (see line 16). To refer to the canvas, you'd need to use `ide.canvas`.

Reply:
Badly named variable, sorry for that. Maybe rename into canvasContainer.

Reply:
I just tested old exports and I can confirm: we moved from "it just works in firefox" to "it works everywhere _except_ for firefox"... So yes, this should get an issue after merging this 🤷.